Sources/Testing/Running/Configuration.TestFilter.swift

```diff
@@ -250,7 +250,7 @@ extension Configuration.TestFilter.Kind {
       return zip(
         lhs.apply(to: testGraph),
         rhs.apply(to: testGraph)
-      ).mapValues(op.functionValue)
+      ).mapValues { op.functionValue($1.0, $1.1) }
```

Comment:
Can we use named args here? I don't normally mind the conciseness but my brain is expecting `.0` to be the key and `.1` to be the value.